tgui/packages/tgui/styles/interfaces/NuclearBomb.scss

```diff
@@ -32,12 +32,6 @@ $background-beige: #e8e4c9;
   }
 }
 
-.NuclearBomb__Button--1 {
-  background-color: #d3cfb7 !important;
-  border-color: #d3cfb7 !important;
-  color: #a9a692 !important;
-}
```

Comment:
It's intentionally discolored, that's why it's a different class

Reply:
![image](https://github.com/tgstation/tgstation/assets/51863163/b5a6a178-153b-45d1-aafb-9efd037126f2)

Reply:
Humm see I was wondering about why its been like that for so long then or why it had its own class. I will set it up so you can have the main device that uses the pad send classes for it instead.  Humm  kind of makes me wonder if I can put random "smudges" or fingerprints on some of the keys.  Let me play with it.

Reply:
![i5hpreOPp6](https://github.com/tgstation/tgstation/assets/1102671/1227613a-2319-495e-a82f-f3e4d718b340)
There we go, the normal safe keypad is on a separate css class.